scipy/interpolate/_fitpack2.py

```diff
@@ -1269,6 +1269,18 @@ class SmoothBivariateSpline(BivariateSpline):
     -----
     The length of `x`, `y` and `z` should be at least ``(kx+1) * (ky+1)``.
 
+    This routine constructs spline knot vectors automatically via the FITPACK
+    algorithm. The spline knots may be placed away from the data points. For
+    some data sets, this routine may fail to construct an interpolating spline,
+    even if one is requested via ``s=0`` parameter. In such situations, it is
```

Comment:
I don't understand this knot construction failure well, but does this happen especially in linear interpolation? If so, I think this information is helpful.
```suggestion
    even if one is requested via ``s=0`` parameter. This might happen especially in linear interpolation (kx=ky=1). In such situations, it is
```

Reply:
TBH I don't understand it either, and I don't know if it's specific for kx=ky=1 or not. So I'm on a fence on this addition. If you feel it's helpful, great, let's add it.

Reply:
> I don't know if it's specific for kx=ky=1
OK, so the general comment is fine. Please ignore my comment.

Reply:
Just to close the loop: indeed, the same warning shows up the example that is now in the tutorial: https://scipy.github.io/devdocs/tutorial/interpolate/smoothing_splines.html#d-smoothing-splines